public/app/features/explore/state/explorePane.ts

```diff
@@ -76,6 +77,45 @@ export function changePanelState(
   };
 }
 
+/**
+ * Tracks the state of correlation helper data in the panel
+ */
+interface ChangeCorrelationHelperData {
+  exploreId: string;
+  correlationEditorHelperData?: ExploreCorrelationHelperData;
+}
+const changeCorrelationHelperDataAction = createAction<ChangeCorrelationHelperData>(
+  'explore/changeCorrelationHelperData'
+);
+
+export function changeCorrelationHelperData(
```

Comment:
Does it need to be a thunk? Could we just make it an action?

Reply:
done!